app/views/templates/nextReturnSection.scala.html

```diff
@@ -38,7 +38,11 @@ <h2 class="heading-medium" style="margin-top: 0px">@messages("returnObligation.n
 
   <p>
     <a id="vat-returns-link" href="@{appConfig.vatReturnDeadlinesUrl}" data-journey-click="return:view-deadlines:vat-details">
-      @messages("returnObligation.viewReturns")
+        @if(isNonMTDfB) {
```

Comment:
Shouldn't this be wrapped in a feature switch?

Reply:
The `isNonMTDfB` is the switch. Is this not how it's done?

Reply:
If the feature switch is set to false, then this value will always be false.